searchlib/src/vespa/searchlib/features/distancetopathfeature.cpp

```diff
@@ -50,9 +49,9 @@ DistanceToPathExecutor::execute(uint32_t docId)
             double len = std::sqrt(len2);
 
             // For each document location, do
-            for (uint32_t loc = 0; loc < _intBuf.size(); ++loc) {
+            for (long loc : _intBuf) {
```

Comment:
The underlying integer type is determined via transitive typedefs; should this instead be `auto` to always be in sync with any changes (unlikely but possible)?

Reply:
Fixed in #25883